### ISSUE TITLE
Plugin not working with Gradle 7 and checkstyle

### ISSUE THREAD

**Issue by ilopmar** [spring-io/nohttp#44]
I have been trying to upgrade some applications that use this plugin to Gradle 7 but it seems there is something missing or wrong. I see that you have released version 0.0.6 that is compatible with Gradle 7, but it is not working for me.

I have a multi-project build structure although for this simple project I only have one module, but for my real application I have more modules. I'm not sure if this will affect or not to the issue.

### To reproduce the issue

- `git clone https://github.com/ilopmar/no-http-app-gradle`
- `cd no-http-app-gradle`
- `./gradlew checkstyleNohttp`

The task fails with:

```
Execution failed for task ':checkstyleNohttp'.
> Cannot add config_loc to checkstyle.configProperties. Please configure the configDirectory on the checkstyle task instead.
```

I've tried setting the following even if it's not necessary because that is the default directory according to https://docs.gradle.org/current/dsl/org.gradle.api.plugins.quality.CheckstyleExtension.html#org.gradle.api.plugins.quality.CheckstyleExtension:configDirectory

```groovy
checkstyle {
    ...
    configDirectory = file("${rootDir}/config/checkstyle")
    ...
}
```

But it doesn't work either. Debugging the plugin I've seen that the path is set correctly:

![image](https://user-images.githubusercontent.com/559192/114420949-147c2c00-9bb5-11eb-9c1d-41f8c032582e.png)

Another thing I've tried is setting that to another directory that doesn't exist like:

```groovy
checkstyle {
    ...
    configDirectory = file("${rootDir}/config/checkstyle_foo")
    ...
}
```

That fails as expected with:
```
  - Type 'Checkstyle' property 'configDirectory' specifies directory '/home/ivan/workspaces/no-http-app-gradle/config/checkstyle_foo' which doesn't exist.
```

And then if I rename the `config/checkstyle` directory to `config/checkstyle_foo` it fails with the first error I showed.

If in this same project I rollback the wrapper to 6.8.3 it works as expected.

**Comment by rwinch** [comment]
@marcphilipp I don't suppose you are able to look at this since you fixed the original issue?

**Comment by marcphilipp** [comment]
I submitted #46 to fix this.

**Comment by rwinch** [comment]
Closing in favor of the PR gh-46

**Comment by rwinch** [comment]
Thanks for the report @ilopmar I have released 0.0.7 which fixes this issue.

**Comment by ilopmar** [comment]
Thanks @rwinch for the new release! :100:
